test/integration/verify/controls/aws_ec2_instance.rb

```diff
@@ -8,19 +8,20 @@
   default: 'default.Example',
```

Comment:
We should cover a successful `it { should_not have_roles }` test.  This would uncover any of the `nil` errors that I'm anticipating.